lib/dslash_pack2.cu

```diff
@@ -2,17 +2,24 @@
 
 // STRIPED - spread the blocks throughout the workload to ensure we
 // work on all directions/dimensions simultanesouly to maximize NVLink saturation
-#define STRIPED
 // if not STRIPED then this means we assign one thread block per direction / dimension
 
+#ifdef NVSHMEM_COMMS
```

Comment:
Style: the direct `NVSHMEM` includes should be hidden in a `shmem_helper.h` file.